pkgs/development/python-modules/fastavro/default.nix

```diff
@@ -1,8 +1,8 @@
-{ buildPythonPackage
+{ lib
+, buildPythonPackage
 , cython
```

Comment:
please keep alphabetical order

Reply:
`lib` is located first for the vast majority of Python modules and the position where `nix-init` puts it.

Reply:
I know it's a matter of style... it's just that my style is to keep things alphabetical :P this change is also not relevant to the real content of the PR, better to keep the git blame clean